### DISCUSSION TITLE
Carreira front end

### DISCUSSION THREAD

**Discussion by marcelocaselli** [balta-io/forum#334] [Tira Dúvidas] [answered]
Boa tarde,

Desde que comecei a programar, me dediquei ao back end, linguagem c#, inclusive estou estudando a carreira back end atualmente.
Porém, tenho observado que a maior parte das vagas exigem que o desenvolvedor conheça algum front end.
Fui começar a ver o treinamento front end, onde o mesmo ja começa com angular. Para realizar o treinamento da carreira front end eu preciso de algum pré-requisito? tipo conhecer java script, html, css?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @marcelocaselli, como vai?

Sim, é necessário conhecer HTML, CSS e JS, mas teremos um conteúdo (Evento) sobre eles em breve.
Te aviso por E-mail
